ruby/example_code/ec2/ec2-ruby-example-create-vpc.rb

```diff
@@ -1,44 +1,87 @@
-# snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
-# snippet-sourceauthor:[Doug-AWS]
-# snippet-sourcedescription:[Creates a VPC with DNS support and hostname enabled and adds a tag to the VPC.]
-# snippet-keyword:[Amazon Elastic Compute Cloud]
-# snippet-keyword:[create_vpc method]
-# snippet-keyword:[Vpc.modify_attribute method]
-# snippet-keyword:[Vpc.create_tags method]
-# snippet-keyword:[Ruby]
-# snippet-sourcesyntax:[ruby]
-# snippet-service:[ec2]
-# snippet-keyword:[Code Sample]
-# snippet-sourcetype:[full-example]
-# snippet-sourcedate:[2018-03-16]
-# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-#
-# This file is licensed under the Apache License, Version 2.0 (the "License").
-# You may not use this file except in compliance with the License. A copy of the
-# License is located at
-#
-# http://aws.amazon.com/apache2.0/
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX - License - Identifier: Apache - 2.0
+
+require 'aws-sdk-ec2'
+
+# Creates a virtual private cloud (VPC) in
+#   Amazon Virtual Private Cloud (Amazon VPC) and then tags

```

Comment:
More extra spaces between the '#' and the start of the comment text. Just looks odd.

Reply:
Agreed. See my earlier comment about this.